example/esy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "@opam/atd": "*",
+    "@opam/atdgen": "*"
+  },
+  "resolutions": {
+    "@opam/atd": "github:ahrefs/atd:atd.opam#6a1f166",
+    "@opam/atdgen": "github:ahrefs/atd:atdgen.opam#6a1f166"
```

Comment:
it should be possible to use the latest release of atd/atdgen now rather than to pin on a commit

Reply:
@Khady fixed in 90d55ca.

I will update main `esy.json` in another PR to keep the scope of this one narrowed down to `example` folder (we'd need to update `bs-platform` too).